### PR TITLE
move optimizely snippet

### PR DESCRIPTION
**Pull request by MatthewPiatetsky**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -55,7 +55,7 @@
     "lodash.pick": "4.4.0",
     "lodash.snakecase": "4.1.1",
     "lodash.without": "4.4.0",
-    "newrelic": "5.11.0",
+    "newrelic": "^5.13.1",
```
Comment (bbaker6225) [inline package.json]: Is this change needed for this?

Comment (MatthewPiatetsky) [reply package.json]: travis failed on me needing to run npm audit fix so i ran npm install and npm audit fix.  
don't really know if that's the correct process though or why the integrity stuff changed. think it might have to do with sha or something?

Comment (bbaker6225) [reply package.json]: Looks like the package `https-proxy-agent` needs to be manually upgraded (based off travis)

Comment (bbaker6225) [reply package.json]: https://github.com/codecov/codecov-node/issues/157 looks like we're waiting on codecov unfortunately
